contracts/src/ethereum/TokenBridge.sol

```diff
@@ -7,6 +7,8 @@ import "@joriksch/sg-contracts/src/starkware/contracts/interfaces/ProxySupport.s
 import "@joriksch/sg-contracts/src/starkware/cairo/eth/CairoConstants.sol";
 import "../../test/IStarknetMessaging.sol";
 
+import "./utils.sol";
```

Comment:
This isn't used?

Reply:
It is, I commented out the lines which use it for testing. Will add them back after I get the results